admiral/pkg/clusters/handler.go

```diff
@@ -31,6 +32,14 @@ type WeightedService struct {
 	Service *coreV1.Service
 }
 
+type K8sObjectMetadata struct {
```

Comment:
Should we move this to: https://github.com/istio-ecosystem/admiral/blob/585c2ea5d9dab5642352f2c8f14794cb1b8006bb/admiral/pkg/controller/common/types.go?

Reply:
Good idea, let me do that.

Reply:
Done